README.md

```diff
@@ -153,6 +153,8 @@ N/A
 
 Contributions, issues, and feature requests are welcome!
 
+Original design idea by Cindy Shin in Behance.
+
```

Comment:
- [ ] I see your amazing presentation that leave me without word :+1: , but the link should not be in the pull request summary but in the README file  according to the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#project-documentation), Kindly create the another section in your README file and drop there your link as you did for live demo link :smile: 
- [ ] Also make sure in your presentation you show each version of your project design. :+1:

Reply:
@iambenkis This is inside the README.md file, am I missing something here? - it is not in a pull request summary, you grabbed its link in like 156 of the readme file

Reply:
@iambenkis You need to clarify on what "each version" exactly is, what version are you talking about?